phpstan.neon.dist

```diff
@@ -4,6 +4,9 @@ parameters:
         - public
         - src
         - test
+    ignoreErrors:
+        - '#Call to static method [a-zA-Z0-9\\_]+\(\) on an unknown class Tideways\\Profiler.#'
+        - '#Class Tideways\\Profiler not found.#'
```

Comment:
Heads up: I'll probably also add @maglnet/composer-require-checker later on. Not relevant to this patch, but the impedance between environments with/without extension is real